tests/test_image_stack.py

```diff
@@ -16,7 +16,8 @@ def setUp(self):
                                            60,    # dim_y = 60 pixels,
                                            2.0,   # noise_level
                                            4.0,   # variance
-                                           2.0 * i)  # time
+                                           2.0 * i,  # time
```

Comment:
should we have a test added for the image_stack class to check that it can properly access individual image psfs?

Reply:
Good call. I added a new test (test_different_psfs) to check that it is handling the per-image PSFs correctly.